specification.md

```diff
@@ -4238,7 +4143,7 @@ Different states now can use this retry strategy defined. For example:
 
 ```yaml
 onErrors:
-- error: Inventory service timeout
+- condition: ${ .item.error == 'Inventory service timeout' }
```

Comment:
same here 

.item | .error == "Inventory service timeout"

and you will need to escape the quotes